rotkehlchen/tests/exchanges/test_coinbase.py

```diff
@@ -849,3 +850,75 @@ def test_coinbase_query_deposit_withdrawals_unexpected_data(function_scope_coinb
         expected_warnings_num=0,
         expected_errors_num=1,
     )
+
+
+def test_asset_conversion():
+    trade_b = {
+        "id": "77c5ad72-764e-414b-8bdb-b5aed20fb4b1",
+        "type": "trade",
+        "status": "completed",
+        "amount": {
+            "amount": "-1000.000000",
```

Comment:
can both amounts be negative here?

Reply:
Do you mean in both trades? In this example USDC and BTC amounts to be negative?